src/mongocrypt-log-private.h

```diff
@@ -51,7 +51,7 @@ _mongocrypt_log_set_fn (_mongocrypt_log_t *log,
                         void *ctx);
 
 
-#ifdef MONGOCRYPT_TRACE
+#ifdef MONGOCRYPT_ENABLE_TRACE
```

Comment:
Is `MONGOCRYPT_ENABLE_TRACE` different from `MONGOCRYPT_TRACE` or is `MONGOCRYPT_TRACE` just being replaced?

If `MONGOCRYPT_TRACE` is being replaced I think `integrating.md` would have to be updated too.

Reply:
Unfortunately, the cmake option is `ENABLE_TRACE`. The C macro is `MONGOCRYPT_ENABLE_TRACE`. And the environment variable to enable it is `MONGOCRYPT_TRACE` lol.

`integrating.md` is right, though because this macro was wrong trace logs weren't being generated.